spec/library_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'yaml'
+require './lib/library.rb'
+
+#collection = YAML.load_file('./lib/booksdb.yml')
```

Comment:
Make sure to delete unused code and unnecessary comments when submitting your work.